apps/src/code-studio/initializeCodeMirror.js

```diff
@@ -54,7 +55,7 @@ module.exports = function (target, mode, callback, attachments) {
     if (previewElement.length > 0) {
       const originalCallback = callback;
       updatePreview = editor => {
-        previewElement.html(marked(editor.getValue()));
+        previewElement.html(marked(editor.getValue(), {renderer: stylelessRenderer}));
```

Comment:
Even though I'm now using the stylelessRenderer on the levelbuilder edit page, it seems like it still lets styles through. They also show up on the level page itself (with and without my change).